src/routes/store/gift-cards/[slug]/+page.svelte

```diff
@@ -30,6 +30,21 @@
     }`;
 
 	let count = 0, selectedDenomination = 0;
+
+	const getAmountWithoutDiscount = (/** @type {number} */ amount = 0, /** @type {number} */ commission_percent = 0) => { 
+
+		let amount_to_pay = amount - ((amount*commission_percent) / 100);
```

Comment:
Whereas this formula here is very correct, the grouping can make it unclear.

```( amount * (commission_percent / 100) );``` will be a better grouping to clearly show that we are explicitly converting commission percentage into a percentage and therefore we require an integer like 5 to be passed in instead of 0.05.